manuscript/teaching-reproducibility.Rmd

```diff
@@ -133,6 +133,35 @@ that are particularly effective for teaching reproducibility successfully:
 In this paper, we will discuss why we believe each of these are important,
 provide examples of how to incorporate these in your teaching.
 
+## Academic setting and background
+
+The specific teaching pedagogies we present in this paper
+have been tested in classrooms of 10-200 learners from varying educational backgrounds,
+including undergraduate students taking their only data science course
+and graduate students looking to specialize in data science.
```

Comment:
```suggestion
and graduate students from diverse academic backgrounds who have completed at least one undergraduate-level university course in computer science, statistics and mathematics who are looking to specialize in data science.
```
Note - that sentence might be too long, so it might be better to split it up. But I think we want to include this context about their prior knowledge.

Reply:
I kept the information below instead since it was easier to fold in the additional MDS details there.